src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -44,6 +47,10 @@ internal HttpInMetricsListener(string name, Meter meter, AspNetCoreMetricsInstru
         this.options = options;
         this.httpServerDuration = meter.CreateHistogram<double>(HttpServerDurationMetricName, "ms", "Measures the duration of inbound HTTP requests.");
 
+        // TODO: This needs to be changed to  "s" (seconds). This is blocked until we can change the default histogram.
+        // See: https://github.com/open-telemetry/opentelemetry-dotnet/issues/4797
+        this.httpServerRequestDuration = meter.CreateHistogram<double>(HttpServerRequestDurationMetricName, "ms", "Measures the duration of inbound HTTP requests.");
```

Comment:
do we create 2 instruments now?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4802#discussion_r1304988605 We should only create 2 if user has opted in for both old and new(feature-freeze) conventions. @TimothyMothra - Could you please check if Java is following the same approach.

Reply:
Looks like Java is emiting one or the other, but not both.
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/c2a3ba02d3ad1d89e332932f0af3665b0afec362/instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpServerMetrics.java#L61-L64

This conflicts with my understanding. Everywhere else we've used the environment variable, we've had to consider the both scenario. I'll touch base with @Trask and get his take.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/59455fc1662415c9b4c1a5aea10f4aac0b9eddc7/specification/metrics/semantic_conventions/http-metrics.md?plain=1#L51

Reply:
> This conflicts with my understanding. Everywhere else we've used the environment variable, we've had to consider the both scenario. I'll touch base with @trask and get his take.

I agree 👍, I've sent this on the Java side: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/9320

(also note https://github.com/open-telemetry/semantic-conventions/pull/278)